src/jsutils/didYouMean.d.ts

```diff
@@ -0,0 +1,4 @@
+export default function didYouMean(
```

Comment:
Please copy comments jsdoc comments for all methods of new stuff if they present in Flow sources.